src/char/char.c

```diff
@@ -1358,106 +1358,116 @@ int char_rename_char_sql(struct char_session_data *sd, uint32 char_id)
 	return 0;
 }
 
-int char_check_char_name(char * name, char * esc_name)
+/**
+* Function that check if a given name is valide within config
+* currently more oriented to for newchar creation
+* @param inout name : Char name to check
+return e_makechar_error
+*/
```

Comment:
I think parameters is to generic. I was refering as config = char_name_option in char_athena.conf
I should also probably specify the other check :
min name len, special character, reserved name and check duplicate